test-ui-test/EXPECTED.txt

```diff
@@ -0,0 +1,73 @@
+____________________________________________________________
```

Comment:
It is great that your expected output is a lot and it tests a lot of the functionalities. However, I feel that it is very long, maybe it is tedious to write this much, might consider lesser input and output in the future so it is not as tedious.